src/broadcasts/interfaces/create-broadcast-options.interface.ts

```diff
@@ -0,0 +1,79 @@
+import type * as React from 'react';
+import type { PostOptions } from '../../common/interfaces';
+import type { RequireAtLeastOne } from '../../common/interfaces/require-at-least-one';
+import type { ErrorResponse } from '../../interfaces';
+
+interface EmailRenderOptions {
+  /**
+   * The React component used to write the message.
+   *
+   * @link https://resend.com/docs/api-reference/broadcasts/sendl#body-parameters
+   */
+  react: React.ReactNode;
+  /**
+   * The HTML version of the message.
+   *
+   * @link https://resend.com/docs/api-reference/broadcasts/sendl#body-parameters
+   */
+  html: string;
+  /**
+   * The plain text version of the message.
+   *
+   * @link https://resend.com/docs/api-reference/broadcasts/sendl#body-parameters
+   */
+  text: string;
+}
+
+interface CreateBroadcastBaseOptions {
+  /**
+   * The name of the broadcast
+   *
+   * @link https://resend.com/docs/api-reference/broadcasts/sendl#body-parameters
+   */
+  name?: string;
+  /**
+   * The id of the audience you want to send to
+   *
+   * @link https://resend.com/docs/api-reference/broadcasts/sendl#body-parameters
+   */
+  audienceId: string;
+  /**
+   * A short snippet of text displayed as a preview in recipients' inboxes, often shown below or beside the subject line.
+   *
+   * @link https://resend.com/docs/api-reference/broadcasts/sendl#body-parameters
+   */
+  previewText?: string;
+  /**
+   * Sender email address. To include a friendly name, use the format `"Your Name <sender@domain.com>"`
+   *
+   * @link https://resend.com/docs/api-reference/broadcasts/sendl#body-parameters
+   */
+  from: string;
+  /**
+   * Reply-to email address. For multiple addresses, send as an array of strings.
+   *
+   * @link https://resend.com/docs/api-reference/broadcasts/sendl#body-parameters
+   */
+  replyTo?: string | string[];
+  /**
+   * Email subject.
+   *
+   * @link https://resend.com/docs/api-reference/broadcasts/sendl#body-parameters
```

Comment:
```suggestion
   * @link https://resend.com/docs/api-reference/broadcasts/send#body-parameters
```

(just realized, should this be `create` instead of send?)